Changelog.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 2.0.5
```

Comment:
Lets keep this in 2.0.4. I'll publish a new release once this is merged in

Reply:
cool, let me rebase then

Reply:
done, should be ready for merging